src/CSF.SRDashboard.Client/Models/FullName.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Models
+{
+    public class FullName
```

Comment:
I'd suggest changing the class to AssigneeName, so that we can changes the Names parameter to FullName
```suggestion
        public class AssigneeName
```